standalone/src/recording/components/recording_view/index.tsx

```diff
@@ -102,7 +106,7 @@ const RecordingView = () => {
         return (
             <div style={style.screenContainer as CSSProperties}>
                 <video
-                    ref={screenPlayerRef}
+                    ref={screenRefCb}
```

Comment:
I think you can use the `setScreenPlayerNode` directly, and then there's no need for the `useCallback`: 
```suggestion
                    ref={setScreenPlayerNode}
```

Reply:
@cpoile Happy to simplify but are we 100% confident? I can run a few more tests but wasn't sure myself so opted for the more elaborate approach.

Reply:
I don't think that's working. What we could do is keep everything as it was and just add the `ref.current` dependency to `useEffect` but I still can't figure out if that's safe to do in our case.

Reply:
> don't think that's working. What we could do is keep everything as it was and just add the `ref.current` dependency to `useEffect` but I still can't figure out if that's safe to do in our case.

That probably won't work, the `ref.current` won't trigger a rerender (that's what the SO question was about, afaict). 
The `ref={setScreenPlayerNode}` should work... But I guess it's not a big deal, just thought it would simplify things. :)